### ISSUE TITLE
Publishing designing contest fails

### ISSUE THREAD

**Issue by jukka-mustonen** [Hankintailmoitukset/hilma-api#117]
Hilma seems to send malformed request to TED.
When attempted to publish designing contest hilma returns TED schema error.
designing contest in question "devel_2686"

The notice cannot be published to TED: The element 'CONTRACTING_BODY' in namespace 'http://publications.europa.eu/resource/schema/ted/R2.0.9/reception' has incomplete content. 
List of possible elements expected: 'ADDRESS_PARTICIPATION_IDEM, ADDRESS_PARTICIPATION, URL_TOOL, CA_TYPE, CA_TYPE_OTHER, CE_ACTIVITY, CE_ACTIVITY_OTHER' in namespace 'http://publications.europa.eu/resource/schema/ted/R2.0.9/reception'.

**Comment by jannef** [comment]
Error message suggest some mandatory data is missing. Please provide all the data.

**Comment by jannef** [comment]
I had some trouble locating the notice body in question, as it was not evident to which environment it was posted to. Now investigating the notice content.

**Comment by jannef** [comment]
Organisation is defined as following in your notice:
```json
"Organisation": {
    "DepartmentIdentifier": null,
    "Information": {
      "OfficialName": "Demolan seudun hankintatoimi",
      "NationalRegistrationNumber": "1088146-2",
      "Department": null,
      "NutsCodes": [
        "FI"
      ],
      "PostalAddress": {
        "StreetAddress": "Postitusosoitekatu 1",
        "PostalCode": "80100",
        "Town": "Joensuu",
        "Country": "FI"
      },
      "TelephoneNumber": "+358 xxxxx",
      "Email": "xxxx@cloudia.fi",
      "ContactPerson": "xxxx",
      "MainUrl": "http://www.demoiluseutu.fi/hankintatoimi",
      "ValidationState": 0
    },
    "ContractingAuthorityType": 4,
    "ContractingType": 0,
    "MainActivity": 1,
    "MainActivityUtilities": 0,
    "OtherContractingAuthorityType": null,
    "OtherMainActivity": null
  }
```

And in the project, the procurement category is defined to be:
```json
"Project": { "ProcurementCategory": 4 }
```

which means the design contest is for utilities (infrastruktuuri ja erityiset alat) governed by directive 2014/25/EU. Which means we are dealing with Contracting Enity (CE), instead of Contractring Authority (CA). The relevant differences between CE and CA for API user is MainActivity/MainActivityUtilities.

Instead of `MainActivity`, `MainActivityUtilities` must be used - this is a different enumeration. Regardless, if "other" is selected in the enumeration, the additional information still goes to `OtherMainActivity` regardless of if we are dealing with CA/CE.

TLDR: Supply MainActivityUtilities and omit MainActivity.
